### PR TITLE
Update CVE issues

### PR DESCRIPTION
**Pull request by jacob-delgado**
Update docs for CVEs released on Feb 22, 2022

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: .spelling [hunk 1]
```diff
@@ -238,15 +239,6 @@ CVE-2019-9513
 CVE-2019-9514
 CVE-2019-9515
 CVE-2019-9518
-CVE-2019-15605
```
Comment (jacob-delgado) [inline .spelling:241]: I just used `sort` on this list

Comment (craigbox) [reply .spelling:241]: could you do a numeric sort, as there's no leading 0?

File: content/en/docs/releases/supported-releases/index.md [hunk 2]
```diff
@@ -74,11 +74,9 @@ Istio does not guarantee that minor releases that fall outside the support windo
...
-| 1.11.x                     | 1.11.7+                              |
-| 1.10.x                     | 1.10.4+                              |
-| 1.9.x                      | 1.9.9                                |
-| 1.8 and earlier            | None                                 |
+| Minor Releases   | Patched versions with no known CVEs |
+|------------------|-------------------------------------|
+| 1.13.x           | 1.13.1+                             |
+| 1.12.x           | 1.12.4+                             |
+| 1.11.x           | 1.11.7+                             |
+| 1.10 and earlier | None, all versions are vulnerable   |
```
Comment (craigbox) [inline content/en/docs/releases/supported-releases/index.md:82]: "have known vulnerabilities" might be better than "are vulnerable"

File: content/en/news/releases/1.12.x/announcing-1.12.4/index.md [hunk 3]
```diff
@@ -18,6 +18,28 @@ This release fixes the security vulnerabilities described in our February 22nd p
 - __[CVE-2022-23635](https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=CVE-2022-23635])__:
   CVE-2022-23635 (CVSS Score 7.5, High):  Unauthenticated control plane denial of service attack.
 
+### Envoy changes
+
+to be transparent.
```
Comment (craigbox) [inline content/en/news/releases/1.12.x/announcing-1.12.4/index.md:23]:
```suggestion
### Envoy CVEs

At this time it is not believed that Istio is vulnerable to these CVEs in Envoy. They are listed, however,
to be transparent.
```

File: content/en/news/releases/1.13.x/announcing-1.13.1/index.md [hunk 4]
```diff
@@ -18,6 +18,29 @@ This release fixes the security vulnerabilities described in our February 22nd p
 - __[CVE-2022-23635](https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=CVE-2022-23635])__:
   CVE-2022-23635 (CVSS Score 7.5, High):  Unauthenticated control plane denial of service attack.
 
+### Envoy changes
```
Comment (craigbox) [inline content/en/news/releases/1.13.x/announcing-1.13.1/index.md:21]:
```suggestion
### Envoy CVEs
```

File: content/en/news/security/istio-security-2022-003/index.md [hunk 5]
```diff
@@ -31,30 +31,42 @@ For simple installations, istiod is typically only reachable from within the clu
...
+- __[CVE-2022-21654](https://github.com/envoyproxy/envoy/security/advisories/GHSA-5j4x-g36v-m283])__:
+  (CVSS Score 7.3, High): Incorrect configuration handling allows mTLS session re-use without re-validation after validation settings have changed.
 
-- __[CVE-2022-21655](https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=CVE-2022-21655])__:
-  CVE-2022-21655 (CVSS Score 7.5, High): Envoy 1.21 and earlier - Incorrect handling of internal redirects to routes with a direct response entry.
+- __[CVE-2022-21655](https://github.com/envoyproxy/envoy/security/advisories/GHSA-7r5p-7fmh-jxpg])__:
+  (CVSS Score 7.5, High): Incorrect handling of internal redirects to routes with a direct response entry.
 
-The following CVE did not apply to Istio 1.11.6.
+The following CVEs only affected Istio 1.12.0-1.12.3 and 1.13.0. It did not affect Istio 1.11.
```
Comment (craigbox) [inline content/en/news/security/istio-security-2022-003/index.md:49]:
```suggestion
The following CVEs only affected Istio 1.12.0-1.12.3 and 1.13.0. They did not affect Istio 1.11.
```

**Review by craigbox — CHANGES_REQUESTED** [review]

File: content/en/news/security/istio-security-2022-003/index.md [hunk 6]
```diff
@@ -31,30 +31,42 @@ For simple installations, istiod is typically only reachable from within the clu
...
 
-- __[CVE-2022-23606](https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=CVE-2022-23606])__:
-  CVE-2022-23606 (CVSS Score 4.4, Moderate): Envoy 1.20 and later - Stack exhaustion when a cluster is deleted via Cluster Discovery Service.
+- __[CVE-2022-23606](https://github.com/envoyproxy/envoy/security/advisories/GHSA-9vp2-4cp7-vvxf])__:
+  (CVSS Score 4.4, Moderate): Stack exhaustion when a cluster is deleted via Cluster Discovery Service.
+
+The following CVEs did not have patches for the version of Envoy used in Istio 1.11. While the CVE exists, Istio
+does not believe that it is affected by this issue.
```
Comment (Kmoneal) [inline content/en/news/security/istio-security-2022-003/index.md:55]: Do the CVEs affect 1.12 and 1.13? A Table may be good for which fixes exist and what versions are effected.

File: content/en/docs/releases/supported-releases/index.md (hunk 2 above)
Comment (jacob-delgado) [reply content/en/docs/releases/supported-releases/index.md:82]: Fixed

File: content/en/news/security/istio-security-2022-003/index.md (hunk 6 above)
Comment (jacob-delgado) [reply content/en/news/security/istio-security-2022-003/index.md:55]: Added a table

File: content/en/news/security/istio-security-2022-003/index.md [hunk 7]
```diff
@@ -31,30 +31,21 @@ For simple installations, istiod is typically only reachable from within the clu
...
+| [CVE-2022-21657](https://github.com/envoyproxy/envoy/security/advisories/GHSA-837m-wjrv-vm5g]) | 3.1, Low      | X.509 Extended Key Usage and Trust Purposes bypass                                                                        | No, next release. | No, next release. | No, next release.                |
 
 ## Am I Impacted?
 
 You are at most risk if you are running Istio in a multi-cluster environment, or if you have exposed your istiod externally.
 
 ## Credit
 
-We would like to thank John Howard (Google) for the report and the fix.
+We would like to thank `Adam Korczynski` ([`ADA Logics`](https://adalogics.com)) and `John Howard` (Google) for the report and the fix.
```
Comment (craigbox) [inline content/en/news/security/istio-security-2022-003/index.md:51]:
```suggestion
We would like to thank Adam Korczynski ([`ADA Logics`](https://adalogics.com)) and John Howard (Google) for the report and the fix.
```

**Review by craigbox — DISMISSED** [review]

**Review by AdamKorcz — APPROVED** [review]
I consent to adding my name, thanks!
LGTM.

**Review by Kmoneal — APPROVED** [review]

**Review by craigbox — APPROVED** [review]

**Pull request merged**